lib/health_check/middleware_health_check.rb

```diff
@@ -0,0 +1,44 @@
+module HealthCheck
+  class MiddlewareHealthcheck
+
+    def initialize(app)
+      @app = app
+    end
+
+    def call(env)
+      uri = env['PATH_INFO']
+      if uri.include? HealthCheck.uri
+        response_type = uri[/\.(json|xml)/,1]
+        response_method = 'response_' + (response_type || 'plain').to_s
+        checks = env['QUERY_STRING'][/checks=([a-z0-9\-]*)/,1] || 'standard'
```

Comment:
how are checks separated in QUERY_STRING ? with coma ? 
question is if REGEX should contain coma